test/transforms.jl

```diff
@@ -685,6 +685,22 @@
     @test t == tₒ
   end
 
+  @testset "Coerce" begin
+    x1 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x2 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x3 = [5.0, 5.0, 5.0, 5.0, 5.0]
+    t = Table(;x1, x2, x3)
+
+    # apply test
+    n, c = apply(Coerce(:x1=>Count, :x2=>Count), t)
+    @test elscitype(n.x1) == Count && elscitype(n.x2) == Count
```

Comment:
Please split this test into two separate tests. Also, I think we don't need to import elscitype, we can test directly the machine type of the resulting table. In this case the Count scitype creates columns with Int. So just check that the eltype of the column is Int.